src/tokens/tokens.scss

```diff
@@ -259,111 +219,56 @@ $button-sizing-icon-xs: 14px;
 $button-sizing-icon-sm: 16px;
 $button-sizing-icon-md: 20px;
 $button-sizing-icon-lg: 22px;
-$input-button-sizing-height-xs: 32px;
-$input-button-sizing-height-sm: 36px;
-$input-button-sizing-height-md: 44px;
-$input-button-sizing-height-lg: 52px;
-$checkbox-radio-sizing-sm: 16px;
-$checkbox-radio-sizing-md: 20px;
-$checkbox-radio-sizing-lg: 24px;
+$inputButton-sizing-height-xs: 32px;
+$inputButton-sizing-height-sm: 36px;
+$inputButton-sizing-height-md: 44px;
+$inputButton-sizing-height-lg: 52px;
 $tag-sizing-height-sm: 22px;
 $tag-sizing-height-md: 26px;
 $spinner-sizing-xs: 14px;
 $spinner-sizing-sm: 16px;
 $spinner-sizing-md: 18px;
 $spinner-sizing-lg: 20px;
-$modal-sizing-max-width-xs: 420px;
-$modal-sizing-max-width-sm: 640px;
-$modal-sizing-max-width-md: 940px;
-$modal-sizing-max-width-lg: 1200px;
+$checkboxRadio-sizing-sm: 16px;
+$checkboxRadio-sizing-md: 20px;
+$checkboxRadio-sizing-lg: 24px;
+$modal-sizing-maxWidth-xs: 420px;
+$modal-sizing-maxWidth-sm: 640px;
+$modal-sizing-maxWidth-md: 940px;
+$modal-sizing-maxWidth-lg: 1200px;
 
 /**
 * @tokens BoxShadow
 * @presenter Shadow
 */
-$ref-box-shadow-xs: (
-  x: 0,
-  y: 2px,
-  blur: 2px,
-  spread: 0,
-  color: rgba(0,0,0,.06),
-  type: dropShadow,
-);
-$ref-box-shadow-sm: (
-  0: [object Object],
-  1: [object Object],
-);
-$ref-box-shadow-md: (
-  0: [object Object],
-  1: [object Object],
-);
-$ref-box-shadow-lg: (
-  x: 0,
-  y: 20px,
-  blur: 50px,
-  spread: 0,
-  color: rgba(0,0,0,.20),
-  type: dropShadow,
-);
-$button-box-shadow-primary: (
-  x: 0,
-  y: 0,
-  blur: 0,
-  spread: 0,
-  color: rgba(0,0,0,0),
-  type: dropShadow,
-);
-$card-box-shadow: (
-  0: [object Object],
-  1: [object Object],
-);
-$tooltip-box-shadow: (
-  0: [object Object],
-  1: [object Object],
-);
-$dropdown-box-shadow: (
-  0: [object Object],
-  1: [object Object],
-);
-$modal-box-shadow: (
-  x: 0,
-  y: 20px,
-  blur: 50px,
-  spread: 0,
-  color: rgba(0,0,0,.20),
-  type: dropShadow,
-);
+$ref-boxShadow-sm: 1px 2px 6px 0 rgba(0,0,0,.11);
```

Comment:
Actually, we've got another small problem 😊 I switched from using tokens that I created specifically for web to tokens that are now shared between web and mobile. There has been some minor changes. Most of them are not relevant. But you can see one of the change in the diff here for shadows.

To be honest, I'm not sure how exactly this happened that in your branch you didn't have the correct config from `master` but you had there already updated compiled tokens 😕 It doesn't matter. Anyway, in the current correct version, shadows are defined as an object or an array of objects (for inter-platform compatibility), for example:

```
{
          "x": "0",
          "y": "2px",
          "blur": "2px",
          "spread": "0",
          "color": "rgba(0,0,0,.06)",
          "type": "dropShadow"
        }
```

or

```
[
          {
            "x": "0",
            "y": "4px",
            "blur": "10px",
            "spread": "0",
            "color": "rgba(0,0,0,.08)",
            "type": "dropShadow"
          },
          {
            "x": "0",
            "y": "2px",
            "blur": "4px",
            "spread": "0",
            "color": "rgba(0,0,0,.04)",
            "type": "dropShadow"
          }
        ]
```

and this need to be transformed to `0 2px 2px 0 rgba(0,0,0,.06)` and `0 4px 10px 0 rgba(0,0,0,.08), 0 2px 4px 0 rgba(0,0,0,.04)` (respectively). There's a helper function for that in `tokens/helpers.tsx` called `getBoxShadow` so you should use that.